src/database/standalone-sqlite.ts

```diff
@@ -526,6 +526,12 @@ export class StandaloneSqliteDatabaseWorker {
 
         this.stmts.bundles.upsertNewDataItem.run({
           ...rows.newDataItem,
+          root_id: item.root_tx_id,
```

Comment:
Overriding these shouldn't be necessary. The call to `dataItemToDbRows` should already be setting them.

Reply:
removed here: [50539d2](https://github.com/ar-io/ar-io-node/pull/136/commits/50539d2beea91b2e4fa9e497b8884c88660dfc5a)